spec/factories/rental.rb

```diff
@@ -9,7 +9,12 @@
   end
 
   factory :invalid_rental, parent: :rental do
+    user_id { create(:user).id }
```

Comment:
You can clean this up by using `association`, as opposed to IDs. Because you already have access to a user factory, this can be written as:

`association :user`

Documentation: https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md